website/components/InstallButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { useCta } from "hooks/use-cta"
+
+const InstallButton: React.FC = () => {
```

Comment:
Why InstallButton component is a full section and not just 
```
 <a href={cta} className="button install-button">
              Start helping
            </a>
```

Reply:
same element as per: eg.:
https://github.com/Altruisto/altruisto/blob/master/website/custom/index/covid.html#L1112

Reply:
Let's separate this into two components:
InstallButton with only the button with cta
something like CallToActionSection with the rest wrapping around InstallButton 

Otherwise the naming is somewhat confusing